src/marqo/vespa/vespa_client.py

```diff
@@ -425,6 +425,20 @@ def delete_batch(self,
 
         return batch_response
 
+    def update_batch(self, batch: List[VespaDocument],
+                     schema: str,
+                     concurrency: int = 10,
+                     timeout: int = 60) -> UpdateBatchResponse:
+
+        if not batch:
```

Comment:
Is this just partial updates? Or will this work for normal updates to? If partial, name should reflect that